packages/FormElement/src/components/Layout/Layout.js

```diff
@@ -0,0 +1,17 @@
+import styled, { css } from "styled-components";
+
+export const LayoutFlexParent = styled.div`
+  display: flex;
+`;
+
+export const LayoutLeftCol = styled.div(
+  ({ width }) => css`
+    width: 180px;
+    ${width && `width: ${width}px;`}
+  `
+);
```

Comment:
Exporting just the styled component like this means the `width` prop isn't documented anywhere (automatically anyway).  I think we should wrap this one in a component with PropTypes.

Also, what about a percentage for `width`? That would allow nice, fluid layouts. Maybe something like `if typeof width === number` add `"px"`?